client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -165,6 +166,18 @@
         this.edit = edit;
         this.editItem = null;
 
+        this.isDeskChanged = function (){
+            return desks.changeDesk;
+        }
+
+        this.highlightsDeskChanged = function (){
+            if(desks.changeDesk) {
+                $location.url('/workspace/monitoring');
```

Comment:
Not sure how to do it, tried to find something similar but didn't find it. Can you give me example?